README.md

```diff
@@ -9,7 +9,7 @@
 <img src="https://user-images.githubusercontent.com/8661268/114321508-64a6b000-9b1b-11eb-95ef-b84c91387cff.png" width="50%">
 </p>
 
-Experimental rust wrapper for the [iceoryx](https://github.com/eclipse-iceoryx/iceoryx) IPC middleware.
+Experimental Rust bindings for the [iceoryx](https://github.com/eclipse-iceoryx/iceoryx) IPC middleware.
```

Comment:
Experimental sounds always like something is most likely to explode. What do you think of a term like "Rust binding in an early stage of development" or "Rust binding where the API may still change"

I know you already merged this but when you create another branch you may adjust this as well

Reply:
This will be part of #4 which will come later today